flume/README.md

```diff
@@ -0,0 +1,80 @@
+# Agent Check: Flume
+
+## Overview
+
+This check monitors [Apache Flume][1].
+
+## Setup
+
+### Installation
+
+To install the Flume check on your host:
+
+
+1. Install the [developer toolkit][2].
+
+ on any machine.
```

Comment:
This line seems lost. Is it part of step 1 or 2, or not supposed to be there at all?

Reply:
Hm that was intended to be:
> 1. Install the developer toolkit on any machine.

I agree that does not read very well however, removing it now 👍